.github/workflows/test.yml

```
@@ -0,0 +1,35 @@
+name: Build
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "22"
```

```suggestion
          node-version: "lts/*"
```

Unless you need a specific node version, using the LTS will prevent future maintenance when node 22 is eventually deprecated for github runners.

---

I personally prefer an explicit version. If we have some bug where CI passes with node 22 but fails with node 24, then we're in a situation where CI is suddenly failing even though we haven't changed anything in our code.

If we make the version explicit, we'll recognize when we bump the node version if that causes a break.

---

I agree with @mtlynch on this one. We almost always stick with explicit node versions on our open source projects and also on my work projects.

---

Fair enough.

> If we make the version explicit, we'll recognize when we bump the node version if that causes a break.

Considering the fact that all of these automated builds were working fine in the past and were removed completely by a new maintainer, I'd be worried about node 22 being deprecated in the future, breaking the CI, and that being used as a reason to remove them all again. I don't expect that to happen but I never expected everything to be removed in v3 either :shrug: